storage/innobase/row/row0import.cc

```diff
@@ -2115,14 +2176,30 @@ dberr_t PageConverter::operator()(buf_block_t* block) UNIV_NOTHROW
 	return DB_SUCCESS;
 }
 
-/*****************************************************************//**
-Clean up after import tablespace. */
-static	MY_ATTRIBUTE((nonnull, warn_unused_result))
+static void reload_fts_table(row_prebuilt_t *prebuilt,
+                             dict_table_t* table)
+{
+
+  /* Reload the table in case of hidden fts column */
+  const table_id_t id= prebuilt->table->id;
+  prebuilt->table->release();
+  dict_sys.remove(prebuilt->table);
+  prebuilt->table= dict_table_open_on_id(
+                      id, true, DICT_TABLE_OP_NORMAL);
+  prebuilt->table->space= table->space;
```

Comment:
Shouldn’t these fields already be equal?
```c++
  ut_ad(prebuilt->table->space == table->space);
```
It would also be good to add `ut_ad(prebuilt->table != table);` at the start of the function, because otherwise the call to `dict_sys.remove()` could free `table`.